consensus/polybft/contractsapi/contractsapi.go

```diff
@@ -238,3 +238,18 @@ func (s *SubmitFunction) EncodeAbi() ([]byte, error) {
 func (s *SubmitFunction) DecodeAbi(buf []byte) error {
 	return decodeMethod(CheckpointManager.Abi.Methods["submit"], buf, s)
 }
+
+type InitializeCheckpointManagerFunction struct {
+	NewBls          types.Address `abi:"newBls"`
+	NewBn256G2      types.Address `abi:"newBn256G2"`
+	NewDomain       types.Hash    `abi:"newDomain"`
+	NewValidatorSet []*Validator  `abi:"newValidatorSet"`
+}
+
+func (I *InitializeCheckpointManagerFunction) EncodeAbi() ([]byte, error) {
+	return CheckpointManager.Abi.Methods["initialize"].Encode(I)
+}
+
+func (I *InitializeCheckpointManagerFunction) DecodeAbi(buf []byte) error {
+	return decodeMethod(CheckpointManager.Abi.Methods["initialize"], buf, I)
+}
```

Comment:
Change generator, so it makes pointer receiver in lower case:

```suggestion
func (i *InitializeCheckpointManagerFunction) EncodeAbi() ([]byte, error) {
	return CheckpointManager.Abi.Methods["initialize"].Encode(i)
}

func (i *InitializeCheckpointManagerFunction) DecodeAbi(buf []byte) error {
	return decodeMethod(CheckpointManager.Abi.Methods["initialize"], buf, i)
}
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1166/commits/0761e2dc8fcade2f8f976bb1976629310e14bacf